single_tech_samples/streamanalytics/streamingjobs.bicep

```diff
@@ -19,9 +28,9 @@ var input = {
     datasource: {
       type: 'Microsoft.Devices/IotHubs'
       properties: {
-        iotHubNamespace: 'iot-${name}-${env}'
-        sharedAccessPolicyName: sharedAccessPolicy.name
-        sharedAccessPolicyKey: sharedAccessPolicy.key
+        iotHubNamespace: iotHubs.name
```

Comment:
nit - could just use the param 'iotHubsName'

Reply:
I think it's the same difference in this case. Just stylistic, unless I'm missign something?